google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddMerchantCenterDynamicRemarketingCampaign.java

```diff
@@ -82,7 +86,8 @@
   }
 
   public static void main(String[] args) throws IOException {
```

Comment:
Is it a must to throw this IOException, did n't see this in other samples.

Reply:
Yes, because a downstream method throws the IOException.